### PR TITLE
Migrate to cb3 and new pinnings.

### PR DESCRIPTION
**Pull request by johanneskoester**
This PR migrates the recipe to the new cb3 based build environment and thereby also updates the pinning scheme to utilize the pinning feedstock.

### CODE REVIEW

**Comment by johanneskoester** [comment]
@conda-forge/core something is wrong with conda-smithy 3.1.6. It has completely destroyed the variant config files, see commit 2ce8673.

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by johanneskoester** [comment]
Better. Any idea why it does not work when doing it locally @isuruf?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,10 +19,12 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }}
+  host:
     - python
     - pip
     - cython
-    - numpy 1.9.*
+    - numpy >=1.9.2,<1.14.0
```
Comment (isuruf) [inline recipe/meta.yaml:27]: See https://conda-forge.org/docs/meta.html#building-against-numpy

**Comment by ebolyen** [comment]
Looks like these are all floating point precision errors. I've created an issue upstream:    https://github.com/biocore/scikit-bio/issues/1587

**Pull request closed without merging**